core/src/test/scala/unit/kafka/zk/ZKEphemeralTest.scala

```diff
@@ -1,177 +0,0 @@
-/**
- * Licensed to the Apache Software Foundation (ASF) under one or more
- * contributor license agreements.  See the NOTICE file distributed with
- * this work for additional information regarding copyright ownership.
- * The ASF licenses this file to You under the Apache License, Version 2.0
- * (the "License"); you may not use this file except in compliance with
- * the License.  You may obtain a copy of the License at
- * 
- *    http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
-*/
-
-package kafka.zk
```

Comment:
Since broker is still using zookeeper, would this test class still be useful?

Reply:
The broker doesn't use `ZkUtils` anymore and hence why I removed the `ZKCheckedEphemeral` code and tests. There is a `KafkaZkClient.CheckedEphemeral` that hasn't been removed and it's still used.